### ISSUE TITLE
[MVP 2 Colors] Adjust colors for better accessibility

### ISSUE THREAD

**Issue by georgewrmarshall** [MetaMask/design-tokens#9]
# Description
A lot of our current color values don't meet AA or AAA contrast accessibility standards with the current combination of font-size and contrast. This should be a hard requirement for all user actions and high priority information components such as buttons, text and alerts. 

# Technical Details
- All colors should follow similar rules to our current brand colors. Where only lightness in (HSL) color modes are changed
- In the screenshots I am using the figma plugin: [A11y - Color Contrast Checker](https://www.figma.com/community/plugin/733159460536249875/A11y---Color-Contrast-Checker) 

# Acceptance Criteria
- All values for the foreground colors(`text.default`,`text.alternative`) against the background colors(`background.default`, `background.alternative`) in both light and dark themes meet AA and AAA accessibility contrast standards
- All values for the subcategories `default` and `inverse` in the user action category primary meet all AA and AAA accessibility standards
- All values for the subcategories `default` and `inverse` in the state categories success, error, warning and info meet all AA and AAA accessibility standards
- There is thorough documentation for colors that don't meet accessibility standards such as muted and disabled and the reasoning for that.

# Screenshots

<img width="1137" alt="Screen Shot 2022-02-05 at 4 19 19 PM" src="https://user-images.githubusercontent.com/8112138/152630491-ca9a7e77-7f41-4dec-8aa4-f8790c532cb3.png">

<img width="954" alt="Screen Shot 2022-02-05 at 4 24 55 PM" src="https://user-images.githubusercontent.com/8112138/152630557-b3dec773-0894-4117-a23b-adb82b002276.png">
